build.gradle

```diff
@@ -41,16 +41,36 @@ allprojects {
         mavenCentral() { metadataSources { mavenPom(); ignoreGradleMetadataRedirection() } }
     }
 
-    sourceCompatibility = 11
-    targetCompatibility = 11
```

Comment:
Do we need to use the jdk plugin instead?

Reply:
We're already using the jdk plugins so removing this was just cleanup

https://github.com/palantir/streams/blob/1bf9d1286cbbd3ea9c41cc7d5bf1a4f5442b75d3/build.gradle#L13-L35